### PR TITLE
Feature/aws bedrock cross region inference

### PR DESCRIPTION
**Pull request by koenvaneijk**
## Title

AWS Bedrock cross-region inference

## Relevant issues

[Feature]: Cross-region inference profile AWS Bedrock #5492

## Type

🆕 New Feature

## Changes

Changed provider determination logic for AWS Bedrock inference

### CODE REVIEW

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
can you add a test for this

**Comment by koenvaneijk** [comment]
added a test if that's okay! :-)

File: litellm/tests/test_bedrock_completion.py [hunk 1]
```diff
@@ -70,6 +70,18 @@ def test_completion_bedrock_claude_completion_auth():
         # Add any assertions here to check the response
         print(response)
 
+        # Second call to test cross-region inference
```
Comment (ishaan-jaff) [inline litellm/tests/test_bedrock_completion.py:73]: Can you use Mock calls / Async Mock calls. See `test_sagemaker`

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
requested change on test

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
can you add an example of this to our docs too

**Comment by koenvaneijk** [comment]
yeah, i'll try to get to it, quite limited on time right now and can't really dive in! if anyone else could pick this up would be awesome 👏

File: litellm/tests/test_bedrock_completion.py (hunk 1 above)
Comment (koenvaneijk) [reply litellm/tests/test_bedrock_completion.py:73]: sounds good, hoping to get to this at a later point! if anyone else gets to this before i do, would be okay 👍🏻

**Comment by ishaan-jaff** [comment]
If you can make the test use mock / magic mock, I can take care of the rest @koenvaneijk

**Comment by ghost** [comment]
See - https://github.com/BerriAI/litellm/issues/5492#issuecomment-2327576384

This already works with litellm for all converse api models (covers anthropic already)

**Comment by ghost** [comment]
Closing pr as feature is already supported. 

@koenvaneijk If you're seeing an error during usage, can you update the original ticket with a reproducible example? I shared how i tested it on there.

**Comment by Manouchehri** [comment]
> This already works with litellm for all converse api models (covers anthropic already)

Yep, added in #5430!

**Pull request closed without merging**